index.html

```diff
@@ -570,7 +570,7 @@ <h2 id="how" class="title is-2">How<a class="anchor-link" href="#how">#</a></h2>
 
                     <p>Please note that the encoding process is lossless. As a matter of fact, you can also use a Python one-liner to decode an encoded diagram:</p>
 
-                    <pre><code class="language-sh">echo "eNpLyUwvSizIUHBXqPZIzcnJ17ULzy_KSanlAgB1EAjQ" | python -c "import sys; import base64; import zlib; print(zlib.decompress(base64.urlsafe_b64decode(sys.stdin.read())))"</code></pre>
+                    <pre><code class="language-sh">echo "eNpLyUwvSizIUHBXqPZIzcnJ17ULzy_KSanlAgB1EAjQ" | python2 -c "import sys; import base64; import zlib; print(zlib.decompress(base64.urlsafe_b64decode(sys.stdin.read())))"</code></pre>
```

Comment:
It might be better to add `.decode('utf8')` to make the command compatible with both Python 2 and Python 3, what do you think?

```suggestion
                    <pre><code class="language-sh">echo "eNpLyUwvSizIUHBXqPZIzcnJ17ULzy_KSanlAgB1EAjQ" | python -c "import sys; import base64; import zlib; print(zlib.decompress(base64.urlsafe_b64decode(sys.stdin.read())).decode('utf8'))"</code></pre>
```

Reply:
That makes much more sense :+1:

What would the changes be required for the encoding? (I'm having trouble trying to get the flip side working, due to my not-great Python experience)